firebase-perf/src/test/java/com/google/firebase/perf/application/FragmentStateMonitorTest.java

```diff
@@ -243,6 +244,21 @@ public void fragmentTraceCreation_dropsTrace_whenFragmentNameTooLong() {
     verify(mockTransportManager, times(0)).log(any(TraceMetric.class), any());
   }
 
+  @Test
+  public void fragmentTraceCreation_addsSpecialAttributeValue_whenFragmentDoesNotHaveParent() {
```

Comment:
Can we name this as `fragmentTraceCreation_NoParentFragment_addsNoParentAttributeValue`?

Reference: go/unit-testing-practices#naming

Reply:
`addsNoParentAttributeValue` could be ambiguous (can sound like not adding anything), which is why I used the wording "special attribute value"

Reply:
The new name you proposed makes sense to me.